LoRaEngine/LoraKeysManagerFacade/ConcentratorCredentialsFunction.cs

```diff
@@ -102,7 +103,10 @@ or InvalidCastException
                                               or InvalidOperationException)
                 {
                     this.logger.LogError(ex, "'{PropertyName}' desired property was not found or misconfigured.", CupsPropertyName);
-                    return new UnprocessableEntityResult();
+                    return new ObjectResult($"Failed to fetch '{CupsPropertyName}' property from device twin")
```

Comment:
By changing the type to `ObjectResult` we lose observability compared to before on the LNS side. For example, in `LoRaDeviceAPIService.FetchStationCredentialsAsync` we only log the `ReasonPhrase` (= Internal Server Error) and the Status Code (= 500), which does not tell us much about what went wrong. I would argue that we should also change the log statements on the client side.

Reply:
Updated the log messages for Credentials and Firmware function usage.